docs/Tutorial_VALMOD.ipynb

```diff
@@ -0,0 +1,1344 @@
+{
```

Comment:
<p>Please add a comment to remind the reader that the sum of <code>\alpha_t</code> is zero from equation (7) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/586/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
<p>Thanks for the suggestion. so, I will add the reminder just before the last two equations.</p>